content/de/docs/concepts/workloads/pods/_index.md

```diff
@@ -0,0 +1,373 @@
+---
+reviewers:
+- erictune
```

Comment:
Remove this, unless you know for sure that the original reviewer also speaks German.